catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageMetacardType.java

```diff
@@ -37,6 +41,8 @@
     public ImageMetacardType() {
         super(NITF, (Set<AttributeDescriptor>) null);
         getDescriptors();
+        IIORegistry.getDefaultInstance()
```

Comment:
The `ImageMetacardType` class is being removed in CAL-87. We will need to deconflict this change.